azure-pipelines.yml

```diff
@@ -0,0 +1,23 @@
+# Learn more: https://aka.ms/yaml
+variables:
+  HOST_DMD_VERSION: 2.084.1
+
+jobs:
+  - job: Windows
+    timeoutInMinutes: 120
+    pool:
+      vmImage: 'vs2017-win2016'
+    variables:
+      VSINSTALLDIR: C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise
+    strategy:
+      matrix:
+        x64:
+          OS: win64
```

Comment:
I think we really should avoid having different names for the same operating system for Windows. It mostly causes extra special casing. But that might be for a separate discussion.

Reply:
Well, I'm still trying out things here and it will be take quite a bit until this is up for review ;-)